### ISSUE TITLE
Brand icon history

### ISSUE THREAD

**Issue by jlnrrg** [simple-icons/simple-icons#5019]
First and foremost I'd like to ask if simple-Icons aims to provide most recent brand icons or brand icons in general.
If the latter is the case, is there a way/ interest in contributing past brand icons?
eg. the gmail logo changed often already.

**Comment by runxel** [comment]
Hi @jlnrrg, thanks for your interest in Simple Icons.
We aim to provide the current icons of important brands. 
I don't think anybody is interested in outdated logos, but of course everybody can just check the history of each icon in this repo – it's plain git after all.

Or do you rather mean icons of brands which no longer are operational, like e.g. Pam-Am?

**Comment by jlnrrg** [comment]
@runxel
I host the simple-Icon set for flutter, which directly build from this current build, where past Information is no longer available. So I could (in theory) imagine someone wanting to build an app where different Version of past Icons are listed. 

Like:
gmail.svg
gmail-2012.svg
gmail-2010.svg

**Comment by runxel** [comment]
I can't really imagine any useful use case for that, but if you're interested in building something like that we encourage you to do so. If we see that there is real demand we might incorporate this project at some point, but I think right now we would have trouble to support this.

**Comment by jlnrrg** [comment]
@runxel ok, maybe my example was confusing, let me try again.
As simple-Icon is supported by [shields.io](https://shields.io/), i could want to specify the Excel Version i am familiar with in my github profile.

Implementation wise it would be more of a guidelines than anything else: everytime a brand is updated, the previous icon gets saved under a different name, thus the latest icon would be excel.svg, but also excel-2010.svg would persist.

**Comment by PeterShaggyNoble** [comment]
Thanks for this suggestion, @jlnrrg. After an internal discussion with the rest of the team over the weekend, we all agree with @runxel that, in general, this isn't something we want to pursue. One major problem would be that we'd open ourselves up to more potential legal issues both from the company itself due to misrepresenting its brand by continuing to include their outdated branding in our library and from other companies that may have legally forced the brand to change its logo or icon due to perceived similarities with another brand. I was just the other day, for example, reading up on a case where Apple forced [these guys](https://www.prepear.com/) to [change their logo](https://logos.fandom.com/wiki/Logopedia) which would be an instance where we definitely wouldn't want to keep the old version.

That being said, we have in the past made exceptions and allowances on a case-by-case basis usually in cases of hardware (e.g., PlayStation) and software (e.g., Windows) when people have made use cases (e.g., showing compatibility or for use with emulators) that warranted their inclusion. Your suggested use case for Excel 2010 could fall into that category, too as, like Windows, the different versions of Office and its programmes could be considered individual brands. So, what I would suggest is you create an issue for each icon or family of icons you need with your use case for each and we'll discuss and decide on each one individually.

**Comment by jlnrrg** [comment]
Thank you for discussing it. The legal side seems to indeed be a major point against it, but great to know that individual Version Brands can be supported (after consultation).
